server_tests/test/clientTests.js

```diff
@@ -1514,7 +1517,7 @@ describe("RUST API TEST SUITE", function() {
     describe("Testing /osc/commands/execute camera._bublCaptureVideo endpoint", function() {
         before(function() {
             if (!isBublcam) {
-                return this.skip();
+                this.skip();
```

Comment:
all of these other `before()`s are the same thing, should be merged as you did with timelapse.